### PR TITLE
[Keyboard] TR90

### PR DESCRIPTION
**Pull request by balloondogcaps**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

This PR adds support for the TR90

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/balloondogcaps/tr90/info.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+    "manufacturer": "Balloondog",
+    "keyboard_name": "balloondogcaps/tr90",
```
Comment (fauxpark) [inline keyboards/balloondogcaps/tr90/info.json:3]:
```suggestion
    "keyboard_name": "TR90",
```

File: keyboards/balloondogcaps/tr90/config.h [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/* Copyright 2021 Balloondog
```
Comment (fauxpark) [inline keyboards/balloondogcaps/tr90/config.h:1]: This file can be deleted.

File: keyboards/balloondogcaps/tr90/rules.mk [hunk 3]
```diff
@@ -0,0 +1 @@
+VIA_ENABLE = yes
```
Comment (fauxpark) [inline keyboards/balloondogcaps/tr90/rules.mk:1]:
```suggestion
```

File: keyboards/balloondogcaps/tr90/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    [0] = LAYOUT( /* Base */
+        KC_1,  KC_2, KC_3,
+        KC_4,  KC_5, KC_6,
+        KC_VOLD,  KC_VOLU, MO(1)
+    ),
+    [1] = LAYOUT(
+        KC_7, KC_8, KC_9,
+        KC_0, KC_MINUS, KC_PLUS,
+        RESET, KC_AT , KC_TRNS
+    ),
```
Comment (fauxpark) [inline keyboards/balloondogcaps/tr90/keymaps/via/keymap.c:28]: Need two more layers, or a config.h with `#define DYNAMIC_KEYMAP_LAYER_COUNT 2`.

File: keyboards/balloondogcaps/tr90/readme.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# balloondogcaps/tr90
+
+![balloondogcaps/tr90](https://imgur.com/a/mFn1Tek)
```
Comment (fauxpark) [inline keyboards/balloondogcaps/tr90/readme.md:3]:
```suggestion
![balloondogcaps/tr90](https://i.imgur.com/GCAZotn.jpeg)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/balloondogcaps/tr90/config.h (hunk 2 above)
Comment (balloondogcaps) [reply keyboards/balloondogcaps/tr90/config.h:1]: According to the VIA documentation I need that file.

Source: https://www.caniusevia.com/docs/configuring_qmk

**Comment by balloondogcaps** [comment]
Hey @fauxpark Ryan, I usually don't bump but can you give me some insights on the review schedule? It has almost been 3 months since the original PR.

**Comment by drashna** [comment]
Thanks!

**Pull request merged**